src/main/java/org/zalando/nakadi/domain/SubscriptionBase.java

```diff
@@ -48,7 +50,8 @@
     public SubscriptionBase() {
     }
 
-    public SubscriptionBase(final SubscriptionBase subscriptionBase) {
+    public SubscriptionBase(final SubscriptionBase subscriptionBase, final String id) {
+        this.setId(id);
```

Comment:
the idea of `SubscriptionBase` is that it have only what user entered during creation of subscription. So it can't really have an id, as we generate it later.

Reply:
added in the commit